annotation/src/main/java/org/cbioportal/genome_nexus/annotation/service/internal/MutationAssessorService.java

```diff
@@ -0,0 +1,79 @@
+package org.cbioportal.genome_nexus.annotation.service.internal;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.web.client.RestTemplate;
+import com.mongodb.DBObject;
+import com.mongodb.util.JSON;
+import org.cbioportal.genome_nexus.annotation.domain.MutationAssessor;
+
+import java.io.IOException;
+import java.util.List;
+
+public class MutationAssessorService
+{
+
+    private static String mutationAssessorURL;
+
+    public void setMutationAssessorURL(String mutationAssessorURL)
```

Comment:
We should get this value from `application.proerties`. Easiest way to do it is to add an annotation for the setter, something like this: `@Value("${mutationAssessor.url}")`